cmd/generate_resources.go

```diff
@@ -3185,16 +3185,33 @@ func filterDateFormat(format string, components []string) string {
 		return ""
 	}
 
+	// Check if this is a month-year combination for special comma handling
+	isMonthYear := false
+	hasMonth := false
+	hasYear := false
+	for _, comp := range components {
+		if comp == "M" {
+			hasMonth = true
+		}
+		if comp == "y" {
+			hasYear = true
+		}
+	}
+	isMonthYear = hasMonth && hasYear
```

Comment:
Do we need to worry about whether there are other components here? Like is this check just to determine if this is a month-year pattern, or is a day-month-year pattern also something where we want to convert commas to spaces?